### PR TITLE
Update py34 -> py37

### PR DESCRIPTION
**Pull request by chenrui333**
relates to https://github.com/juju/charm-tools/pull/523

### CODE REVIEW

File: docs/envlist.rst [hunk 1]
```diff
@@ -15,19 +15,19 @@ Configure the Python versions to test with in ``.travis.yml``:
 
 .. code-block:: yaml
 
-    sudo: false
```
Comment (chenrui333) [inline docs/envlist.rst]: No need to have `sudo: false` as it has been consolidated at Travis end.

**Comment by chenrui333** [comment]
https://docs.travis-ci.com/user/reference/xenial/#python-support
xenial build env does not support py2.6.

**Comment by gaborbernat** [comment]
Just drop python 2.6 👍

**Comment by gaborbernat** [comment]
Now we just need to fix the tests failing 👍

**Comment by chenrui333** [comment]
that is right 😄

**Comment by ryanhiebert** [comment]
The thinking behind this is good, though not urgent. Let's keep dropping py34 as a separate issue. And I delayed this enough that it really should be adjusting to 3.8. I'm going to close it, but new PRs dealing with this are welcome. Thank you, @chenrui333 !

**Comment by chenrui333** [comment]
I will find some time to take another look on this, sorry for the long delay. :)

**Comment by ryanhiebert** [comment]
The delay was me, not you, so the apology is mine. Anyway, I finally got to some stuff in this repo, but this ended up not cleaning up as easily as the other PRs, so I concluded that it would be best to go ahead and let you figure out what of it still applied how.

**Pull request closed without merging**